__tests__/app.test.js

```diff
@@ -277,7 +277,6 @@ describe.only("PATCH /api/reviews/:review_id", () => {
             votes: expect.any(Number),
```

Comment:
We dont expect any number, we expect the original vote count PLUS the votes we added